### DISCUSSION TITLE
Borders & Tailwind CSS

### DISCUSSION THREAD

**Discussion by stivaugoin** [mantinedev/mantine#807] [Q&A] [answered]
Hey guys I have problems to add border with Tailwind's classes and I don't know if this is a bug on Mantine or it's just me who doesn't understand well how it's manage internally.

When I add `border-r border-red-500` on whatever component or a simply `div`, nothing append. To see my border I need to add `border-solid` but with this, all side have now border. By default with Tailwind, we don't need to provide a border style. And we don't have `border-r-solid`.

The only way I succeed to have my right border, it's to add `style: {{ borderRightStyle: "solid" }}` or using `sx` props.

Does Mantine apply anything special on the borders? I saw nothing about that on the doc and in the code base.

**Comment by rtivital** [comment]
No, Mantine does not include any global styles for borders. Some components may include border styles, for example, TextInput or Autocomplete. If you need to override those styles, you can use Styles API. Some components have styles that may require `!important` to override.

**Comment by stivaugoin** [comment, marked as answer]
Thanks for your answer @rtivital.

I found the problem and it's not about Mantine and neither a bug. It's the correct behavior when we turn off `preflight` on Tailwind CSS. https://github.com/tailwindlabs/tailwindcss/issues/6979

Sorry for inconvenience 😃
